quesma/queryparser/query_parser_test.go

```diff
@@ -213,212 +213,6 @@ func TestQueryParserNoAttrsConfig(t *testing.T) {
 	}
 }
 
-// TODO this will be updated in the next PR
```

Comment:
I remove 2 tests: 1 is a trivial histogram, another one is complex, but it's already in our aggregation test suite.